miasm2/arch/aarch64/sem.py

```diff
@@ -730,6 +793,36 @@ def irbloc_fix_regs_for_mode(self, irbloc, mode=64):
                 irs[i] = self.expr_fix_regs_for_mode(e)
         irbloc.dst = self.expr_fix_regs_for_mode(irbloc.dst)
 
+    def mod_pc(self, instr, instr_ir, extra_ir):
+        "Replace PC by the instruction's offset"
+        cur_offset = m2_expr.ExprInt64(instr.offset)
+        for i, expr in enumerate(instr_ir):
```

Comment:
This is exactly the same code snippet that line 807, you could move it to a protected method